content/en/docs/contribute/localization.md

```diff
@@ -183,22 +198,28 @@ To collaborate on a development branch:
 
 3. Approvers review and merge feature branches into the development branch.
 
-4. Periodically, an approver merges the development branch to its source branch.
+4. Periodically, an approver merges the development branch to its source branch by opening and approving a new pull request. Be sure to [squash the commits](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors) before approving the pull request.
```

Comment:
@zacharysarah I think the link is helpful but it explains about creating co-authored commit while the readers may expect it contains how to squash the commits.